generate_prometheus_targets.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Create all output directories.
+for project in mlab-sandbox mlab-staging mlab-oti ; do
+  mkdir -p gen/${project}/prometheus/{legacy-targets,blackbox-targets}
+done
+
+# All testing sites and machines.
+SELECT_mlab_sandbox=".*[a-z]{3}[0-9]t.*"
+
+# All mlab4's and the set of canary machines.
+SELECT_mlab_staging=".*(mlab4.[a-z]{3}[0-9]{2}.*"
+SELECT_mlab_staging+="|.*(mlab1.atl06|mlab1.dfw01|mlab1.lhr01|mlab1.nuq02).*"
```

Comment:
I've added a file `plsync/canary_machines.txt`. How does this look?

If this works, then I'll create an issue explaining that this is a bug to be fixed in the new platform or with an attribute in sites.py.